src/backend/services/logger.py

```diff
@@ -1,4 +1,17 @@
 import logging
+import time
+
+from starlette.middleware.base import BaseHTTPMiddleware
+from starlette.requests import Request
+
+
+class LoggingMiddleware(BaseHTTPMiddleware):
+    async def dispatch(self, request: Request, call_next):
+        start_time = time.time()
+        response = await call_next(request)
+
+        logging.info(f"{request.method} {request.url.path} {request.headers}")
```

Comment:
Nit: add newlines after each thing

Reply:
i put method and path on same line, and then header on next line. looks weird otherwise i feel.

<img width="925" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/146760070/6aa9958a-b829-4c2a-8cd0-3ce70edceac2">

vs 

<img width="915" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/146760070/a3ad42a5-7e89-4849-babb-e67966e8a6d2">